src/site/content/en/blog/hands-on-portals/index.md

```diff
@@ -0,0 +1,203 @@
+
+<img class="w-screenshot" src="HTMLPortalElement.png" alt="A screenshot of the DevTools console showing the HTMLPortalElement">
+
+Let’s walk through a basic example.
+
+```javascript
+// Create a portal with the wikipedia page, and embed it
+// (like an iframe). You can also use <portal> tag instead.
+portal = document.createElement('portal');
+portal.src = 'https://en.wikipedia.org/wiki/World_Wide_Web';
+portal.style = '...';
+document.body.appendChild(portal);
+
+// When the user touches the preview (embedded portal):
+// do fancy animation, e.g. expand …
+// and finish by doing the actual transition
+portal.activate();
+```
+
+It’s that simple. Try this code in the DevTools console, the wikipedia page should open up. If you wanted to build something like we showed at Chrome Dev Summit, the following snippet will be of interest.
+
+<img class="w-screenshot" src="portal-preview-demo.gif" alt="A gif of preview portal style demo">
+
+```javascript
+// Adding some styles with transitions
+const style = document.createElement('style');
+const initialScale = 0.4;
+style.innerHTML = `
+  portal {
+    position:fixed;
+    width: 100%;
+    height: 100%;
+    opacity: 0;
+    transition:
+      transform 0.4s,
+      bottom 0.7s,
+      left 0.7s,
+      opacity 1.0s;
+    box-shadow: 0 0 20px 10px #999;
+    transform: scale(${initialScale});
+    bottom: calc(20px + 50% * ${initialScale} - 50%);
+    left: calc(20px + 50% * ${initialScale} - 50%);
+    z-index: 10000;
+  }
+  .portal-reveal {
+    transform: scale(1.0);
+    bottom: 0px;
+    left: 0px;
+  }
+  .fade-in {
+    opacity: 1.0;
+  }
+`;
+const portal = document.createElement('portal');
+// Let’s navigate into the WICG Portals spec page
+portal.src = 'https://wicg.github.io/portals/';
+portal.addEventListener('click', evt => {
+  // Animate the portal once user interacts
+  portal.classList.add('portal-reveal');
+});
+portal.addEventListener('transitionend', evt => {
+  if (evt.propertyName == 'bottom') {
+    // Activate the portal once the transition has completed
+    portal.activate();
+  }
+});
+document.body.append(style, portal);
+
+// Waiting for the page to load.
+// using setTimeout is a suboptimal way and it’s best to fade-in
+// when receiving a load complete message from the portal via postMessage
+setTimeout(_ => portal.classList.add('fade-in'), 2000);
+```
+
+It is also easy to do feature detection to progressively enhance a website using Portals.
+
+```javascript
+if ('HTMLPortalElement' in window) {
+  // If this is a platform that have Portals...
+  const portal = document.createElement('portal');
+  ...
+}
+```
+
+If you want to quickly experience what Portals feel like, try using [uskay-portals-demo.glitch.me](https://uskay-portals-demo.glitch.me). Be sure you access with Chrome Canary and turn on the experimental flag!
```

Comment:
+1 to Rob's suggestion to make this a proper glitch embed. It would be a very cool thing for developers to be able to play with the code a bit in the glitch.

Reply:
As I commented in Robs feedback, I also understand having glitch embed is nice, but at the same time this feature requires Chrome Canary with an experimental flag on which means most of the readers will see an app that doesn't work (although I have a fallback message). So I wast thinking of keeping this as is for now. WDYT?

Reply:
If you're in a rush I think it's ok to land it as-is. But I do think adding the Glitch would be pretty cool. In the above example you're linking to a Glitch and telling the user to turn the feature flag on. What you could do in a follow up revision is embed the Glitch, and use the feature detection steps to detect if the user has Portals enabled, and if not, show them a link to the feature flag.

Likely this post will be up for a long time so when Chrome ships Portals it'll be cool to have a running example in the page that people can edit.